src/views/Housing/index.jsx

```diff
@@ -1,9 +1,22 @@
-import { Card, CardContent, CardHeader, Container, Grid, Typography } from '@mui/material';
+import { Card, Grid } from '@mui/material';
+import { useAuthGroups } from 'hooks';
+import { AuthGroup } from 'services/auth';
+import RDView from './components/RDView';
 
-const Housing = () => (
-  <Grid Container>
-    <Card></Card>
-  </Grid>
-);
+const Housing = () => {
+  const isFaculty = useAuthGroups(AuthGroup.Faculty);
+
+  if (isFaculty) {
```

Comment:
If it does work then this would be `isFacStaff` instead I think

Again not sure if it does work but worth a try? If not then we can use the auth groups